### ISSUE TITLE
Remove custom skin function

### ISSUE THREAD

**Issue by zAfLu** [jonitaikaponi/ckSurf#73]
Hi,

maybe you have notice that some server accounts are banned in the last few days. One of the reasons are "Custom Models  & Skins". Maybe you should remove the custom model function from cksurf to prevent server owners from gettings banned. 

> 
http://blog.counter-strike.net/index.php/server_guidelines/

> – Interfering with systems that allow players to correctly access their own
> CS:GO inventories, items, or profile.
> *[To clarify: it is also not acceptable to provide players with custom
> models and/or weapon skins that do not exist in the CS:GO ecosystem]

**Comment by marcowmadeira** [comment]
I'm pretty sure ckSurf doesn't have any custom model/skin.

**Comment by zAfLu** [comment]
you can add custom models for the replay bots so yes ck has custom model features :)

**Comment by jonitaikaponi** [comment]
True. Valves rules on pure player models is still somewhat vague, but it's
still worth it to disable the option untill we get an absolute clear answer
on them IMO.
30.1.2016 0.36 "zAfLu" <notifications@github.com> kirjoitti:
